### DISCUSSION TITLE
Abandoned project ?

### DISCUSSION THREAD

**Discussion by throrin19** [histoire-dev/histoire#772] [General]
Hello,

Just to ask what everyone is thinking: Is this project abandoned? 

It has not been maintained since April. The vue, vite and other dependencies have been updated but Histoire doesn't seem to care.

Now I can't get Histoire to work because of several vite plugins (adding comlink, sentry, ...) and I can't find a solution here.

I'm thinking of switching to Storybook, but Histoire is starting to waste my time having to debug it every 4 mornings to try and launch it...

**Comment by jclaveau** [comment]
As you can see here, no https://github.com/histoire-dev/histoire/pull/770

**Reply by troyere** [reply]
> As you can see here, no #770

That's only the minimum, there is no new bugfixes, or new features

**Reply by throrin19** [reply]
The MR is merged but there is no new publication on NPM :/

**Reply by jclaveau** [reply]
For what I saw using Quasar, updating to Vite 6 is not easy at all

**Reply by rowanfuchs** [reply]
there are an alarming amount of pull requests open, that does worry me.

**Comment by jpengelbrecht** [comment]
Seems like it might be. Its quite sad, it was a great solution.

**Comment by djurdjen** [comment]
Last release was almost a year ago... so seems more and more like it.
We're currently migrating to Storybook because of the increasing dependency issues. 
Such a shame because it had some real great potential!

**Reply by hugoattal** [reply]
Last release was last month on npm 😉 (Added Vite 6 support)
https://www.npmjs.com/package/histoire

It is not abandoned, it's just not on high development.

**Reply by throrin19** [reply]
@hugoattal So much so that I had to remove the story if I wanted to keep my project up to date 😞 .

**Reply by rowanfuchs** [reply]
> @hugoattal So much so that I had to remove the story if I wanted to keep my project up to date 😞 .

If you want to keep your project up to date, start contributing to this project instead of complaining.

**Reply by jclaveau** [reply]
The fact there is no recent Github release, contrary to NPM, is misleading here
![image](https://github.com/user-attachments/assets/d757fc0d-060c-4d7c-a9f1-3c47605605c1)

**Reply by hmt** [reply]
@jclaveau Just click on the releases and you see the 1.0 pre release. It's there, it works. Please be nice to the people that work for free.

**Reply by Codermar** [reply]
I agree not to get on to people writing OSS and I appreciate and support them. What I don't get is why not reach out or form a close trusted collaborators that maybe interested in reviewing and helping the project move. This is true not just for this project but others.

**Comment by 50rayn** [comment]
Not abandoned — `1.0.0-beta.1` shipped 7 January 2026 and is the current `latest` on npm. Slow, one maintainer, but alive.

If you need Nuxt 4, Svelte 5, SvelteKit or Quasar working today, I maintain [poveste](https://poveste.dev/) — a drop-in fork, same stories and config unchanged.
